src/Xamarin.Android.Build.Tasks/Tasks/ResolveLibraryProjectImports.cs

```diff
@@ -373,5 +403,16 @@ static string GetTargetAssembly (ITaskItem assemblyName)
 				jars.Add (f);
 			}
 		}
+
+		bool SkipResourceProcessing (AssemblyDefinition assembly)
+		{
+			if (assembly.HasCustomAttributes) {
+				foreach (var att in assembly.CustomAttributes) {
+					if (att.AttributeType.FullName == "Android.SkipAndroidResourceProcessingAttribute")
```

Comment:
We might want to just compare `Name`, if we want this to work the same as `[assembly:LinkerSafeAttribute]`: https://github.com/mono/linker/blob/44199a2c62ff221f92c33cbb4ee3e35588adaf4a/tuner/Mono.Tuner/CustomizeActions.cs#L47

This would give the option to add this to the support libraries, and not have to build with a new (not yet released) Xamarin.Android.